kymatio/scattering2d/tests/test_torch_scattering2d.py

```diff
@@ -11,14 +11,14 @@
 backends = []
```

Comment:
Don't think we need to rename this script (at least not yet).

Ideally, we'd like to test all the frontends in the same way, so some clever way or parametrizing the tests with respect to this will be needed.

Reply:
Curretnly, we want to maintain this testing script for making sure that the introduced frontend does not change anything to the documentation and the highlevel use of advanced users.

Later, we would like a minimal script to test the frontend, and we would like to separate backend and frontend tests. Also, we will need to avoid being redundant with torch, and probably to use torch set of testing routines.(maybe there is like in `sklearn` a routine `test_user_module()` or similar) **Very soon, we would like to ensure that the multi-GPU works with the current frontend without any additional changes.**

Reply:
> Very soon, we would like to ensure that the multi-GPU works with the current frontend without any additional changes.

that seems like a good goal and fire test for this approach

Reply:
Ok so we'll just copy this file for each frontend for now? Seems like a mess...

Reply:
for now, let's maintain the test. In a future PR(on the new branch `kymatio-dev`), we'll have to do a serie of test for the `frontend`, `core` and `backend` folders. I think it's the simplest.